lib/invitation.js

```diff
@@ -0,0 +1,15 @@
+const apiRequest = require('./apiRequest');
+
+async function findByInviter() { 
+  const apiResponse = await apiRequest("GET", "/invitation/find-by-inviter", "", null)
+  return apiResponse;
+}
+
+async function createInvitation(country, phone, amount) { 
+  // {\n\t\"country\": \"Italy\",\n\t\"phone\": \"379189****\",\n\t\"amount\": 5000\n}"
```

Comment:
you forgot to remove commented code

Reply:
@agebeyaw  I added them on purpose to provide a template for the request object, if users want to check it. Should I remove them?